vagrant/base.sh

```diff
@@ -63,7 +63,7 @@ get_kafka() {
     scala_version=$2
 
     kafka_dir=/opt/kafka-$version
-    url=https://s3-us-west-2.amazonaws.com/kafka-packages-$version/kafka_$scala_version-$version.tgz
+    url=https://s3-us-west-2.amazonaws.com/kafka-packages/kafka_$scala_version-$version.tgz
```

Comment:
Is this change OK? looks like the structure of kafka-packages on S3 changed at some point. This  URL is the same form I've seen for 1, 1.1, 2.0, and trunk.

\cc @aayars  @maxzheng  @ijuma @ewencp

Reply:
I remembered @mjsax looked at this part once, could you comment?

Reply:
I did not change this. `-$version` was removed here: https://github.com/apache/kafka/pull/3454

Maybe never back ported? Seems ok to me.